### PR TITLE
🚀  Release v1.9.0-rc.0

### PR DESCRIPTION
**Pull request by vishalanarase**
**What this PR does / why we need it**:

Add release note for v1.9.0-rc.0

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

Part Of: https://github.com/kubernetes-sigs/cluster-api/issues/11092

/area release

### CODE REVIEW

File: CHANGELOG/v1.9.0-rc.0.md [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+- k8s.io/code-generator: v0.30.3 → v0.31.2
+- k8s.io/component-base: v0.30.3 → v0.31.2
+- k8s.io/klog/v2: v2.120.1 → v2.130.1
+- k8s.io/kms: v0.30.3 → v0.31.2
+- k8s.io/utils: b307cd5 → 18e509b
+- sigs.k8s.io/apiserver-network-proxy/konnectivity-client: v0.30.0 → v0.30.3
+- sigs.k8s.io/controller-runtime: v0.18.4 → v0.19.1
+
+### Removed
+- github.com/antlr/antlr4/runtime/Go/antlr/v4: [8188dc5](https://github.com/antlr/antlr4/tree/runtime/Go/antlr/v4/8188dc5)
```
Comment (chandankumar4) [inline CHANGELOG/v1.9.0-rc.0.md:430]: Could you remove this link, as it's not found

**Comment by chrischdi** [comment]
Some are missing, at least the following:

- https://github.com/kubernetes-sigs/cluster-api/pull/11277
- https://github.com/kubernetes-sigs/cluster-api/pull/11473
- and some dependabot bumps merged this morning

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Comment by Sunnatillo** [comment]
/hold
We will unhold close to release cut

**Pull request merged**